### ISSUE TITLE
Add time type arg support

### ISSUE THREAD

**Issue by pahanini** [roistat/go-clickhouse#20]
Now arg of type time in query function args will be replaced with ''

e.g clickhouse.NewQuery("Select * From event Where time > ?", time.Now()) will fail

**Comment by m1nor** [comment]
Can't you pass it as string?

```
clickhouse.NewQuery("Select * From event Where time > '?'", time.Now().String())
```

**Comment by pahanini** [comment]
@minor yes, we can use `time.Format()` but time type is frequently used type so it produces a lot of additional code. E.g. 

`clickhouse.NewQuery(sql, startDate.Format(Fmt), endDate.Format(Fmt))`

Instead of 

`clickhouse.NewQuery(sql, startDate, endDate)`

**Comment by isage** [comment]
Accepted pull request actually makes things worse, because instead of converting time to DateTime it converts to Date, losing time part.

**Comment by isage** [comment]
Moreover, accepted pull request marshals time as unquoted string, which is wrong.
